tools/scylla-nodetool.cc

```diff
@@ -89,9 +152,36 @@ const std::map<std::string_view, std::string_view> option_substitutions{
     {"-pw", "--password"},
     {"-pwf", "--password-file"},
     {"-pp", "--print-port"},
+    {"-st", "--start-token"},
+    {"-et", "--end-token"},
 };
 
 const std::map<operation, operation_func> operations_with_func{
+    {{"compact",
+            "Force a (major) compaction on one or more tables",
+R"(
+Forces a (major) compaction on one or more tables. Compaction is an optimization
+that reduces the cost of IO and CPU over time by merging rows in the background.
+
+By default, major compaction runs on all the keyspaces and tables. Major
+compactions will take all the SSTables for a column family and merge them into a
+single SSTable per shard. If a keyspace is provided, the compaction will run on
+all of the tables within that keyspace. If one or more tables are provided as
+command-line arguments, the compaction will run on all tables.
```

Comment:
The last sentence is confusing - I assume that if these one or more tables are provided as command-line arguments, the compaction will run on **these** tables - not on **all** tables.

Reply:
Yes, I coped this text from the website, I guess this should be fixed there too.

Reply:
Fixed in v4.